examples/simple-example/simple-example-core/src/main/java/io/miragon/miranum/integrations/example/application/service/SendMessageService.java

```diff
@@ -25,12 +25,12 @@ public class SendMessageService implements SendMessageUseCase {
             id = "send-message",
             type = "sendMessage",
             appliesTo = {BPMNElementType.BPMN_SERVICE_TASK},
-            version = 0.1)
+            version = 01)
     public Answer sendMessage(final SendMessageCommand message) {
         log.info("Received message: " + message);
 
-        messageApi.correlateMessage(new CorrelateMessageCommand(message.getName(), message.getKey(), Map.of()));
+        messageApi.correlateMessage(new CorrelateMessageCommand(message.getContent(), message.getKey(), Map.of()));
```

Comment:
We should create an overload to the correlateMessageCommands constructor to not take variables. Saw this Map.of() at multiple places